ska_dbi/sqsh.py

```diff
@@ -0,0 +1,93 @@
+import subprocess
+from pathlib import Path
+from astropy.table import Table
+
+SYBASE = "/soft/SYBASE16.0"
+ENV_VARS = [
+    f"SYBASE={SYBASE}",
+    "LD_LIBRARY_PATH=/soft/SYBASE16.0/OCS-16_0/lib:/soft/SYBASE16.0/OCS-16_0/lib3p64:/soft/SYBASE16.0/OCS-16_0/lib3p",
+]
+SQSH_BIN = "/usr/local/bin/sqsh.bin"
+
+
+class Sqsh(object):
+    def __init__(
+        self,
+        server=None,
+        user=None,
+        passwd=None,
+        database=None,
+        sqshrc=None,
+        authdir="/proj/sot/ska/data/aspect_authorization",
+        **kwargs,
+    ):
+        DEFAULTS = {"server": "sqlsao", "user": "aca_ops", "database": "axafapstat"}
```

Comment:
This is copied from the sybase portions of ska_dbi.DBI .  I know Tom hates DRY, but my original thought had been to *move* this here from ska_dbi (thus it would not be DRY).  If we decide we want to maintain the sybase interface in ska_dbi it is a different question about if this stuff should be move to shared configuration.

I suppose DEFAULTS already wasn't a global and had bad case.